File: DSA/Prabhat_500102136/Day6/Day6_Q2.java
import java.util.*;
public class Day6_Q2 {
    public static boolean IsFullyIncreasing(int arr[],int n){
        for(int i=0;i<n-1;i++){
            if(arr[i]>=arr[i+1]){
                return false;
            }
        }
        return true;
    }
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter the size of array");
        int n = scan.nextInt();
        System.out.println("Start entering elements of array");
        int arr[] = new int[n];
        for(int i=0;i<n;i++){
            arr[i]= scan.nextInt();
        }
        System.out.println("Array is  "+(IsFullyIncreasing(arr,n)?"Fully Increasing":"Not Fully Increasing"));
   
    }
}

File: DSA/Vedansh_590014816/Day6q2.cpp
#include <iostream>
#include <vector>
using namespace std;
bool check(vector<int>&arr){
    int start=1;
    int prev=0;
    while(start<arr.size()){
       if(arr[prev]>=arr[start]){
           return false;
       }
       start++;
       prev++;
    }
    return true;
}


int main() {
    int n;
    cout << "Enter the number of elements: ";
    cin >> n;

    vector<int> arr(n);
    cout << "Enter the elements: ";
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
    }
  if(check(arr)) cout<<"Yes the array is in increasing order";
  else cout<<"No the array is not in increasing order";


    return 0;
}
